lib/widgets/matrix.dart

```diff
@@ -186,6 +189,11 @@ class MatrixState extends State<Matrix> with WidgetsBindingObserver {
     return candidate;
   }
 
+  Future<void> logout() async {
+    await client.logout();
+    await hiveCollectionToMDatabase.clear();
```

Comment:
what if `hiveCollectionToMDatabase` is not available ?

Reply:
because it was init in `main`?